src/Model/Term.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @file src/Model/Term
+ */
+namespace Friendica\Model;
+
+use dba;
+
+class Term
+{
+	/**
+	 * @param integer $itemid item id
+	 * @return void
+	 */
+	public static function createFromItem($itemid)
+	{
+		$messages = dba::select('item', ['guid', 'uid', 'id', 'edited', 'deleted', 'file', 'parent'], ['id' => $itemid], ['limit' => 1]);
```

Comment:
The only needed fields are `uid, file` and `deleted` can be checked in the condition.